index.html

```diff
@@ -1,278 +1,375 @@
-                                    </ul> 
-                                </section>
-                              </footer>
+            <li class="skill-container bg-red">
+              <h3 class="skill-heading">Frameworks</h3>
+              <ul>
+                <li class="skill-item">Boostrap</li>
+                <li class="skill-item">Ruby on Rails</li>
+                <li class="skill-item">RSpec</li>
+                <li class="skill-item">Capibara</li>
+                <li class="skill-item">Selenium</li>
+              </ul>
+            </li>
+
+            <li class="skill-container bg-yellow">
+              <h3 class="skill-heading">Skills</h3>
+              <ul>
+                <li class="skill-item">Codekit</li>
+                <li class="skill-item">Codepen</li>
+                <li class="skill-item">GitHub</li>
+                <li class="skill-item">GitLab</li>
+                <li class="skill-item">Terminal</li>
+              </ul>
+            </li>
+            <li class="about-background" alt="background-6"></li>
+          </ul>
+        </div>
+      </div>
+
+      <footer class="desktop-v">
+        <section>
+          <img src="assets/icons/image_geometry_1.svg" alt="" />
+          <h4 class="get-started">Get started</h4>
+          <p class="get-paragraph get-para">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </section>
+        <section class="form-container">
+          <form action="https://formspree.io/f/xwkjrzvw" method="post">
+            <ul class="section-form">
+              <li>
+                <label class="label-1">your name</label>
+                <input
+                  class="name-text"
+                  type="text"
+                  id="name"
+                  name="user-email"
+                  placeholder="Thierry..."
+                  required
+                  maxlength="30"
+                />
+              </li>
+              <li>
+                <label class="label-1">your email</label>
+                <input
+                  class="email-border"
+                  type="text"
+                  id="email"
+                  name="email"
+                  placeholder="noumbuht@"
+                  required
+                />
+              </li>
+              <li>
+                <label class="label-1">your Message</label>
+                <textarea
+                  class="textarea1 textarea2"
+                  id="message"
+                  placeholder="Enter your message here..."
+                  required
+                  maxlength="500"
+                ></textarea>
+              </li>
+            </ul>
+            <div class="start-container">
+              <button type="button" class="Start-collaboration">
+                <p>Start collaboration</p>
+              </button>
+            </div>
+          </form>
+        </section>
```

Comment:
> - [ ] I noticed that your Contact form section on the desktop screen does not match the Figma design as much as possible. It would be highly appreciated if you fix that.👍 This would ensure a better user experience.😊
> 
> Figma design:
> 
> ![6-1](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/de288f03-57f6-4f0b-ad29-07579e45a161)
> 
> Your design:
> 
> ![6](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/4d1b764c-4065-4501-96d1-6f4d45db49a1)
>